README.md

```diff
@@ -75,7 +75,7 @@ const client = new smartcar.AuthClient({
 // Redirect to Smartcar's authentication flow
 app.get('/login', function(req, res) {
 
-  const link = client.getAuthUrl({state: 'MY_STATE_PARAM'});
+  const link = client.getAuthUrl();
```

Comment:
why change this?

Reply:
@gurpreetatwal Jack was confused with this when building his first app because it was different than the Doc Center

Reply:
kk, ideally should be in a separate pr, don't want to conflate the changes